src/server_side_loaders/client_side_only_packages.js

```diff
@@ -0,0 +1,3 @@
+const CLIENT_SIDE_ONLY_PACKAGES = /\b(c3|d3|d3v4|react-select|tribute)\b/i;
```

Comment:
Should `d3v4` be sneaking in here?

Reply:
Oh I see, it's because now it has `\b` after, whereas before it matched on `d3` – ignore me!